### PR TITLE
fix/tr-888/right-to-left-review-panel

### PR DESCRIPTION
**Pull request by marpesia**
**Related** to [TR-888](https://oat-sa.atlassian.net/browse/TR-888)

**Description**:
Implement mode RTL on the Review panel.

**Changes**:
Set CSS to align items on RTL mode

**How to test**:
- [ ] Add RTL language:'ar-arb'
- [ ] Add the Review panel on the test creation

### CODE REVIEW

**Review by btamas — APPROVED** [review]

**Comment by btamas** [comment]
**Review Checklist**
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [ ] New code is respecting code style rules
- [ ] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [ ] Acceptance criteria are respected
- [ ] Pull request title and description are meaningful
- [ ] Pull request's target is not `master`
- [ ] Changelog is updated according to changes (if applicable)
- [ ] Documentation is updated according to changes (if applicable)

File: scss/inc/rtl/_review-panel.scss [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+.runner {
+    .test-runner-scope {
+        .test-sidebar.test-sidebar-left {
+            .qti-panel.qti-navigator {
+                .qti-navigator-label {
+                    .icon-up, .icon-down {
+                        margin-left: 0;
+                        margin-right: auto;
+                    }
+                    .qti-navigator-counter {
```
Comment (meefox) [inline scss/inc/rtl/_review-panel.scss:14]: It is not necessary to have specificity for those classes 0.10.1 = 0.2.1 (_rtl.scss) + 0.8.0 (_review-panel.scss)
Screenshot 2021-06-08 at 18 41 57
I woud suggest to keep it lower as much is possible.

Or is there is a reason for this?

**Review by meefox — CHANGES_REQUESTED** [review]

**Review by jsconan — APPROVED** [review]
Looks ok.

image

**Review Checklist**
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [x] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful
- [x] Pull request's target is not `master`
- [x] Commits are following [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/)
- [x] Commits messages are meaningful
- [x] Commits are atomic
- [ ] Changelog is updated according to changes (if applicable)
- [ ] Documentation is updated according to changes (if applicable)

**Review by lecosson — APPROVED** [review]

**Review by meefox — APPROVED** [review]
All good, thanks for fixing this.
Screenshot 2021-06-10 at 12 49 18

**Review Checklist**
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [x] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful
- [x] Pull request's target is not `master`
- [x] Commits are following [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/)
- [x] Commits messages are meaningful
- [x] Commits are atomic
- [ ] Changelog is updated according to changes (if applicable)
- [ ] Documentation is updated according to changes (if applicable)

**Pull request merged**